.vscode/launch.json

```diff
@@ -22,6 +22,8 @@
             "request": "launch",
             "args": [
                 "--runInBand"
+                // To run a single test, add something like so
```

Comment:
isn't it same as "Pyright jest current file" config below?

Reply:
Oh I didn't notice that one. Maybe? Seems like it's missing the '-i' though.

Reply:
Yeah that works. I'll remove the comment.